packages/elements/src/components/ui/buttons/input-field/InputFieldButton.module.css

```diff
@@ -0,0 +1,49 @@
+.inputFieldButton {
+  --swui-inputfieldbutton-width: 24px;
+  --swui-inputfieldbutton-height: 24px;
+  --swui-inputfieldbutton-margin: 4px;
+  --swui-inputfieldbutton-padding: 0;
+  --swui-inputfieldbutton-border: none;
+  --swui-inputfieldbutton-border-radius: 2px;
+  --swui-inputfieldbutton-background-color: transparent;
+  --swui-inputfieldbutton-background-color-disabled: transparent;
+  --swui-inputfieldbutton-icon-color: var(--lhds-color-blue-600);
+  --swui-inputfieldbutton-icon-color-disabled: var(--swui-text-disabled-color);
+  --swui-inputfieldbutton-hover-color: 2px;
+
+  --current-background-color: var(--swui-inputfieldbutton-background-color);
+  --current-icon-color: var(--swui-inputfieldbutton-icon-color);
+
+  background-color: var(--current-background-color);
+  border: var(--swui-inputfieldbutton-border);
+  border-radius: var(--swui-inputfieldbutton-border-radius);
+
+  width: var(--swui-inputfieldbutton-width);
+  height: var(--swui-inputfieldbutton-height);
```

Comment:
Height is fixed `24px` here. The height should probably grow with the input field?